test/zdtm/static/rseq00.c

```diff
@@ -51,7 +51,10 @@ static inline void unregister_glibc_rseq(void)
 	rseq->cpu_id = RSEQ_CPU_ID_REGISTRATION_FAILED;
 
 	/* unregister rseq */
-	syscall(__NR_rseq, (void *)rseq, __rseq_size, 1, RSEQ_SIG);
+	unsigned int size = __rseq_size;
```

Comment:
Would you be able to move the declaration for `unsigned int size;` at the beginning of the `unregister_glibc_rseq()` function?
```
.701 rseq00.c:54:9: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
8.701    54 |         unsigned int size = __rseq_size;
8.701       |         ^~~~~~~~
```